Source/WebKit/WebProcess/GPU/graphics/RemoteGraphicsContextGLProxy.cpp

```diff
@@ -178,6 +178,18 @@ void RemoteGraphicsContextGLProxy::initialize(const String& availableExtensions,
         m_requestableExtensions.add(extension.toString());
 }
 
+std::tuple<GCGLenum, GCGLenum> RemoteGraphicsContextGLProxy::drawingBufferTextureTargetAndBinding()
+{
```

Comment:
as a stylistic detail:
These values never change. 
So the alternative to always querying this with a sync message would be to have initialization structure:
```
RemoteGraphicsContextGLInitializationState {
   String availableExtensions;
   String requestableExtensions;
#if PLATFORM(COCOA)
    uint32_t externalImageTarget;
    uint32_t externalImageBindingQuery;
#endif
}
```